themes/custom/convivial_bootstrap/components/slideshow/_slideshow.scss

```diff
@@ -194,6 +194,10 @@
   padding: 0;
 }
 
+.paragraph--type--slide {
```

Comment:
Need to be merged, which is also dependent on another PR that has not been merged:
https://github.com/morpht/convivial/pull/34